src/makeInfinityAdder.js

```diff
@@ -5,6 +5,22 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+  const adder = (number = undefined) => {
+    if (number === undefined) {
```

Comment:
The check for `number === undefined` is correct for determining when to return the accumulated sum. Ensure that this logic aligns with the intended use case.